src/plugins/dm.c

```diff
@@ -159,6 +159,8 @@ gboolean bd_dm_is_tech_avail (BDDMTech tech, guint64 mode UNUSED, GError **error
  *
  * Returns: whether the new linear mapping @map_name was successfully created
  * for the @device or not
+ *
+ * Tech category: %BD_DM_TECH_MAP-%MD_DM_TECH_MODE_CREATE_ACTIVATE
```

Comment:
This should be `BD_DM_TECH_MODE_*` (same applies for other functions here and in `dm.api`).